src/services/servers/servers.js

```diff
@@ -0,0 +1,24 @@
+import { getFromLocalStorage } from "../../utils/localStorage/localStorage";
+
+const url = "http://playground.tesonet.lt/v1/servers";
+
+const fetchServers = async () => {
+  const token = getFromLocalStorage("token");
```

Comment:
adding token key to localstorage manually allows me to access the home screen with no auth. Might not be bad, but smth to be aware of.

Reply:
aha. galvojau behavior padaryt, kad jeigu server fetch failino del blogo token, tai automatiskai logout padaryt. bet realiai tai vistiek data negausi be token, tai nzn ar svarbu